datasetinsights/estimators/faster_rcnn.py

```diff
@@ -373,27 +371,25 @@ def log_metric_val(self, label_mappings, epoch):
         for metric_name, metric in self.metrics.items():
             result = metric.compute()
             logger.debug(result)
-            mean_result = np.mean(
-                [result_per_label for result_per_label in result.values()]
-            )
-            logger.info(f"metric {metric_name} has mean result: {mean_result}")
-            self.writer.add_scalar(f"val/m{metric_name}", mean_result, epoch)
-
-            self.kfp_writer.add_metric(name=metric_name, val=mean_result)
+            logger.info(f"metric {metric_name} has result: {result}")
+            if metric_name.startswith("m"):
```

Comment:
[mustfix] This assumes knowledge "mXX" are single value metric. Can you directly add this attribute in `EvaluationMetric` class? In this way it's very explicit and does not break the pipeline if users decide to change their metric name alias from `mAP` to `MAP` or something else.

Reply:
Okay. Will add it as an attribute in `EvaluationMetric` class.